### ISSUE TITLE
Crash when pressing unload in game

### ISSUE THREAD

**Issue by pacapelep** [spirthack/CSGOSimple#64]
When I press the unload button in the main screen it works perfectly, but when I start an offline game and press load, a read access violation occurs at line 1192 in imgui_draw.cpp.

**Comment by xAkiraMiura** [comment]
also occurs for me. seems to happen whenever a player dies however this could be coincidence

**Comment by pacapelep** [comment]
yeah i think you're right unfortunately. It seems to be somewhat random. Maybe some of the signatures aren't working any longer but i don't know enough about it to fix it

**Comment by anarh1st47** [comment]
@Vladislavik36 we need an imgui expert here
